python/python/glide/async_commands/core.py

```diff
@@ -3555,14 +3555,35 @@ async def object_freq(self, key: str) -> int:
             key (str): The key of the object to get the logarithmic access frequency counter of.
 
         Returns:
-            int: If `key` exists, returns the logarithmic access frequency counter of the object stored at `key` as an
+            Optional[int]: If `key` exists, returns the logarithmic access frequency counter of the object stored at `key` as an
                 integer. Otherwise, returns None.
 
         Examples:
             >>> await client.object_freq("my_hash")
                 2  # The logarithmic access frequency counter of "my_hash" has a value of 2.
         """
         return cast(
-            int,
+            Optional[int],
             await self._execute_command(RequestType.ObjectFreq, [key]),
         )
+
+    async def object_idletime(self, key: str) -> Optional[int]:
+        """
+        Returns the time in seconds since the last access to the value stored at `key`.
```

Comment:
Does this need to be included?
`The command is only available when the maxmemory-policy configuration directive is not set to one of the LFU policies.`

Reply:
Hmm good question. We don't have that documentation in Java but it might be helpful, not sure. If we want to update with this I will also have to update for object freq

Reply:
Such specific details are usually omitted